modules/media/ajax/FileUpload.php

```diff
@@ -130,6 +130,7 @@ function uploadFile()
     checkDateTaken($dateTaken);
 
     $fileName  = preg_replace('/\s/', '_', $_FILES["file"]["name"]);
+    $fileName  = str_replace("%22", "\"", $fileName);
```

Comment:
does this mean you are accepting quotes in the name of the file ? also its a bit clearer if you use `'"'` in the second argument

Reply:
Well I wanted to include the double quotes because right now we can add a file with it in the name. However we need this check fo it to work. Otherwise I could also send an error through swal refusing the quotes, but in that case what else are we not allowing?

Reply:
`%22` is URI encoding, your $filename is never coming from the URI... I'm not clear on why its being "decoded" (and BTW decoding should be done with `urldecode()` https://www.php.net/manual/en/function.urldecode.php not manually like this)

Reply:
I changed it to urldecode but I cant find where the fileName is being encoded...